MODULE.bazel

```diff
@@ -0,0 +1,17 @@
+module(
+    name = "grpc-web",
+    version = "1.6.0.rc1",
+    compatibility_level = 1,
+    repo_name = "com_github_grpc_grpc_web",
+)
+
+bazel_dep(name = "protobuf", version = "23.1", repo_name = "com_google_protobuf")
```

Comment:
we were using 27.1 in our WORKSPACE file. Is it possible to match it?

Reply:
Done.